api/resources/gene_information.py

```diff
@@ -126,6 +139,200 @@ def get(self, species="", gene_id=""):
             return BARUtils.error_exit("There are no data found for the given gene")
 
 
+@gene_information.route("/genes_by_position/<string:species>/<string:chromosome>/<string:startParam>/<string:endParam>")
+class GeneTAIR10_GFF3(Resource):
+    @gene_information.param("species", _in="path", default="arabidopsis")
+    @gene_information.param("chromosome", _in="path", default="0")
+    @gene_information.param("startParam", _in="path", default=3000)
+    @gene_information.param("endParam", _in="path", default=6000)
+    def get(self, species="", chromosome="", startParam="", endParam=""):
+        """This end point provides genes given position."""
+
+        # Check if all parameters are provided
+        if not chromosome or not startParam or not endParam:
+            return BARUtils.error_exit("Missing parameters"), 400
+
+        # Check if the start param is smaller than end param
+        if startParam >= endParam:
+            return BARUtils.error_exit("Start location should be smaller than the end location")
+
+        # Escape input
+        species = escape(species)
+        chromosome = escape(chromosome)
+        startParam = escape(startParam)
```

Comment:
Add data validation

Reply:
@TeresasaZ , startParam and endParam need to validate digits, to prevent SQL injection. Use this [method](https://github.com/BioAnalyticResource/BAR_API/blob/dev/api/utils/bar_utils.py#L222).